web/src/app/lists/PaginatedList.js

```diff
@@ -147,7 +148,8 @@ export class PaginatedList extends React.PureComponent {
         title: p.string.isRequired,
         subText: p.string,
         isFavorite: p.bool,
-        icon: p.element,
+        icon: p.element, // renders a list item icon (or avatar)
+        avatar: p.element, // renders a list item avatar with a backdrop
```

Comment:
No longer used?

Reply:
Yep, should just be `icon` now. I'll remove